### DISCUSSION TITLE
traefik fails to access the correct container

### DISCUSSION THREAD

**Discussion by albjeremias** [Tecnativa/doodba#398] [Q&A]
Hi,
great stuff you did in here :) 

Well, im trying to setup a gateway server (gw1), for other two servers (web1, web2)... And Im trying to setup this configuration:

gw1 -> traefik (receives all traffic from the internet, and redirects to the correct server)
web1 <- runs a docker-socket-proxy so that traefik on gw1 can connect to this server
web2 <- running other software without docker

So the problem is that traefik shows on the DEBUG log, trying to "proxy" to an invalid ip address which is a valid ip inside the docker containers on the web1 server, but not on the gw1 docker ... so the question is.. is this even possible with this software? Or do I have to go through the docker swarm path? :)

thanks,

**Comment by yajo** [comment]
>  web2 <- running other software without docker

If you want to use traefik to proxy non-docker software, you have to use a different dynamic configuration provider. For example [the file one](https://doc.traefik.io/traefik/providers/file/).

But this question belongs more to the traefik forums than here. 🤷🏼‍♂️
